cobbler/items/distro.py

```diff
@@ -163,6 +163,8 @@ def kernel(self, kernel: str):
         """
         if not isinstance(kernel, str):
             raise TypeError("kernel was not of type str")
+        if not kernel:
+            return
```

Comment:
This is a silent return and it does not change the value of the property. If this has a good cause please explain, otherwise please drop it. Same goes for the `initrd` below.

Reply:
I did not exercise this path. My intention is to allow clearing of the variable, like `cobbler distro edit --initrd ''`.